crates/javy/src/json.rs

```diff
@@ -3,16 +3,16 @@ use crate::serde::{de::Deserializer, ser::Serializer};
 use anyhow::Result;
 
 /// Transcodes a byte slice containing a JSON encoded payload into a [Value].
-pub fn transcode_input<'js>(context: Ctx<'js>, bytes: &[u8]) -> Result<Value<'js>> {
-    let mut deserializer = serde_json::Deserializer::from_slice(bytes);
+pub fn parse<'js>(context: Ctx<'js>, bytes: &mut [u8]) -> Result<Value<'js>> {
+    let mut deserializer = simd_json::Deserializer::from_slice(bytes)?;
```

Comment:
Would this need to be gated by a check for whether the feature that includes the `simd_json` dependency is enabled or not?